manifests/m/Mondago/GoIntergrator/3.2.1.18541.22072/Mondago.GoIntergrator.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mondago.GoIntergrator
+PackageVersion: 3.2.1.18541.22072
+PackageLocale: en-US
+Publisher: Mondago
+PackageName: GoIntegrator

```

This will have to match Control Panel if there is a way to perform a silent install
```suggestion
PackageName: Go Integrator
```